app/employee-app/backend/modules/authorization/authorization.bal

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// 
+import ballerina/jwt;
+import ballerina/log;
+import ballerina/http;
```

Comment:
it seems like the imports are not formatted(ordered) properly. lets use the Ballerina formatter to fix this in all the files

Reply:
Done and resolved.